Public/Src/Tools/SymbolDaemon/SymbolDaemon.cs

```diff
@@ -0,0 +1,424 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Diagnostics.ContractsLight;
+using System.Linq;
+using System.Reflection;
+using System.Threading.Tasks;
+using BuildXL.Ipc.Common;
+using BuildXL.Ipc.ExternalApi;
+using BuildXL.Ipc.Interfaces;
+using BuildXL.Storage;
+using BuildXL.Utilities.CLI;
+using BuildXL.Utilities.Tasks;
+using Microsoft.VisualStudio.Services.Symbol.App.Core.Tracing;
+using Microsoft.VisualStudio.Services.Symbol.WebApi;
+using Newtonsoft.Json;
+using Tool.ServicePipDaemon;
+using static BuildXL.Utilities.FormattableStringEx;
+
+namespace Tool.SymbolDaemon
+{
+    /// <summary>
+    /// Daemon responsible for handling symbol-related requests.
+    /// </summary>
+    public sealed class SymbolDaemon : ServicePipDaemon.ServicePipDaemon, IDisposable, IIpcOperationExecutor
+    {
+        private const string LogFileName = "SymbolDaemon";
+        private const int s_servicePointParallelism = 200;
+

```

Comment:
any reason why it is 200

Reply:
DropDaemon used this value :)

---
In reply to: [314217163](https://github.com/microsoft/BuildXL/pull/727#discussion_r314217163) [](ancestors = 314217163)